### PR TITLE
Get mappings view api service wrappers

### PR DESCRIPTION
**Pull request by amsiglan**
### Description
In this PR:
- Field mappings are fetched from the server and displayed in the Configure field mappings page
    - Added FieldMapping service to facilitate the fetching of mappings from server.
    - Added logic to validate the created mappings to ensure one alias is only mapped to one index field

### Issues Resolved
Partially implements https://github.com/opensearch-project/security-analytics-dashboards-plugin/issues/18

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by AWSHurneyt — APPROVED** [review]

File: public/pages/CreateDetector/components/ConfigureFieldMapping/containers/ConfigureFieldMapping.tsx [hunk 1]
```diff
@@ -43,12 +55,70 @@ export default class ConfigureFieldMapping extends Component<
 
   getAllMappings = async () => {
     this.setState({ loading: true });
-    await this.setState({ mappingsData: EXAMPLE_FIELD_MAPPINGS_RESPONSE });
+    // const mappingsView = await this.props.filedMappingService.getMappingsView(
```
Comment (paladin-tech) [inline public/pages/CreateDetector/components/ConfigureFieldMapping/containers/ConfigureFieldMapping.tsx:58]: Commented code, should it be removed?

**Review by paladin-tech — APPROVED** [review]

File: public/pages/CreateDetector/components/ConfigureFieldMapping/containers/ConfigureFieldMapping.tsx (hunk 1 above)
Comment (amsiglan) [reply public/pages/CreateDetector/components/ConfigureFieldMapping/containers/ConfigureFieldMapping.tsx:58]: good catch, must have changed this for local testing, let me revert this

Comment (amsiglan) [reply public/pages/CreateDetector/components/ConfigureFieldMapping/containers/ConfigureFieldMapping.tsx]: fixed

**Review by sbcd90 — APPROVED** [review]

File: public/pages/CreateDetector/components/ConfigureFieldMapping/containers/ConfigureFieldMapping.tsx [hunk 2]
```diff
@@ -43,12 +55,70 @@ export default class ConfigureFieldMapping extends Component<
...
+    //   this.props.detector.inputs[0].input.indices[0],
+    //   this.props.detector.detector_type
+    // );
+    // if (mappingsView.ok) {
+    //   this.setState({ mappingsData: mappingsView.response });
+    // }
```
Comment (phestecAMZN) [inline public/pages/CreateDetector/components/ConfigureFieldMapping/containers/ConfigureFieldMapping.tsx]: nit: remove commented code?

Comment (phestecAMZN) [inline public/pages/CreateDetector/components/ConfigureFieldMapping/containers/ConfigureFieldMapping.tsx]: nit: remove commented code?

Comment (amsiglan) [reply public/pages/CreateDetector/components/ConfigureFieldMapping/containers/ConfigureFieldMapping.tsx]: Did before merging

**Pull request merged**